src/common/Testimonial/TestimonialSection.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from 'react';
+import TestimonialCard from './TestimonialCard';
+import { fetchTestimonialsHomePage } from 'common/services/request/query/fetch-testimonials';
+import { submit } from 'common/services/request';
+
+// Import Swiper React components
+import { Swiper, SwiperSlide } from 'swiper/react';
+
+// Import Swiper styles
+import 'swiper/css';
+import 'swiper/css/navigation';
+import 'swiper/css/pagination';
+
+// import required modules
+import { Keyboard, Autoplay, Navigation, Pagination } from 'swiper';
+
+function TestimonialSection() {
+  const [testimonials, setTestimonials] = useState([]);
+
+  const fetchTestimonials = async () => {
+    const res = await submit(fetchTestimonialsHomePage());
+    setTestimonials(res);
+  };
+
+  useEffect(() => {
+    fetchTestimonials();
```

Comment:
We don't need all the testimonials to be fetched here. It will make the home page slow... Just fetch recent 5 or 10

Reply:
@atapas It's fetching 9 testimonials. 


query>fetch-testimonials.js
```
export const fetchFiltered = {
  display: 'testimonials',
  name: 'test',
  function: 'testimonials',
  write: false,
  limit: 9,
  return: [
    'id',
    'quote',
    'title',
    'created_at',
    { user_id_map: ['avatarUrl', 'id', 'displayName', 'email'] },
    { testimonials_event: ['id', 'name', 'description'] }
  ],
  orderBy: {
    created_at: 'desc'
  }
};

export function fetchTestimonialsHomePage() {
  return { ...fetchFiltered };
}
```